Cubical/Core/Glue.agda

```diff
@@ -45,44 +45,50 @@ open import Agda.Builtin.Cubical.Glue public
 
 open isEquiv public
 
-fiber : ∀ {ℓ ℓ'} {A : Set ℓ} {B : Set ℓ'} (f : A → B) (y : B) → Set (ℓ-max ℓ ℓ')
+private
+  variable
+    ℓ ℓ' : Level
+    A : Set ℓ
+    B : Set ℓ'
+
+fiber : ∀ {A : Set ℓ} {B : Set ℓ'} (f : A → B) (y : B) → Set (ℓ-max ℓ ℓ')
```

Comment:
I guess it might be because you need to quantify over arbitrary A and B here and not use the A and B from the top variables? If that's the case then I think it would be bit nicer to only use `variable` for the Levels and not A and B.

Reply:
Because if `A` is generalized, the level of A and `l` will become separated variables, causing some problems.

Reply:
I see. Let's remove A and B from the variables and have them explicitly then. It's a bit confusing if we generalize A and B at the top and then still have to abstract over A's and B's in some of the definitions.

Reply:
Ok.

Reply:
Done.

Reply:
Is it? I might be confused, but A and B are still in the variable list?